test/export/test_export.py

```diff
@@ -268,6 +268,42 @@ def forward(self, x, y):
         inp = ([torch.ones(1, 3)], torch.ones(1, 3))
         self._test_export_same_as_eager(f, inp)
 
+    def test_no_tensor_computation(self):
+        class Module(torch.nn.Module):
+            def forward(self, x, y):
+                return y
+
+        f = Module()
+        inp = ([torch.ones(1, 3)], 1)
+        self._test_export_same_as_eager(f, inp)
+
+    def test_no_tensor_computation_2(self):
+        class Module(torch.nn.Module):
+            def forward(self, x, y):
+                return x
+
+        f = Module()
+        inp = (torch.randn(3), 1)
+        self._test_export_same_as_eager(f, inp)
+
+    def test_no_tensor_computation_3(self):
+        class Module(torch.nn.Module):
+            def forward(self, x, y):
+                return 5
+
+        f = Module()
+        inp = (2, 1)
+        self._test_export_same_as_eager(f, inp)
+
+    def test_no_tensor_computation_4(self):
+        class Module(torch.nn.Module):
+            def forward(self, x, y):
+                return x
+
+        f = Module()
+        inp = ([torch.randn(3)], 1)
+        self._test_export_same_as_eager(f, inp)
```

Comment:
it would be nice to use [assertExpectedInline](https://github.com/pytorch/pytorch/blob/b019f38fdd97957f0e5923961ea7cf5b63c0761b/test/export/test_export.py#L4065-L4073) to check that the code matches what we expect

Reply:
added now